xml/System.Diagnostics/ActivityCreationOptions`1.xml

```diff
@@ -143,8 +145,8 @@
         <ReturnType>System.Collections.Generic.IEnumerable&lt;System.Collections.Generic.KeyValuePair&lt;System.String,System.Object&gt;&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets the tags associated with the activity.</summary>
```

Comment:
>Gets the tags associated with the activity. [](start = 17, length = 43)

Gets the tags that the the activity will be created with.

Reply:
```suggestion
        <summary>Gets the tags that the activity will be created with.</summary>
```